lms/djangoapps/course_api/blocks/transformers/milestones.py

```diff
@@ -0,0 +1,100 @@
+"""
+Milestones Transformer
+"""
+
+from django.conf import settings
+
+from edx_proctoring.api import get_attempt_status_summary
+from edx_proctoring.models import ProctoredExamStudentAttemptStatus
+from openedx.core.lib.block_structure.transformer import BlockStructureTransformer, FilteringTransformerMixin
+from util import milestones_helpers
+
+
+class MilestonesTransformer(FilteringTransformerMixin, BlockStructureTransformer):
+    """
+    Exclude proctored exams unless the user is not a verified student or has
```

Comment:
I believe we should change this, given our latest requirements regarding timed exams and hiding content after due dates, etc.

We should change the code so it always hides Proctored exams for non-staff users, regardless of whether they are verified or whether they declined to take the exam.